modules/cluster-logging-collector-log-forward-project.adoc

```diff
@@ -21,65 +21,57 @@ To configure forwarding application logs from a project, you must create a `Clus
 .Example `ClusterLogForwarder` CR
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: instance <1>
-  namespace: openshift-logging <2>
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
+  managementState: Managed
   outputs:
-   - name: fluentd-server-secure <3>
-     type: fluentdForward <4>
-     url: 'tls://fluentdserver.security.example.com:24224' <5>
-     secret: <6>
-        name: fluentd-secret
-   - name: fluentd-server-insecure
-     type: fluentdForward
-     url: 'tcp://fluentdserver.home.example.com:24224'
-  inputs: <7>
+   - name: <output_name> #<1>
+     type: <output_type> #<2>
+     #...
+  inputs: #<3>
    - name: my-app-logs
      application:
         namespaces:
-        - my-project <8>
+        - my-project #<4>
   pipelines:
-   - name: forward-to-fluentd-insecure <9>
-     inputRefs: <10>
+   - inputRefs: #<5>
      - my-app-logs
-     outputRefs: <11>
+     name: forward-to-fluentd-insecure #<6>
+     outputRefs: #<7>
      - fluentd-server-insecure
      labels:
```

Comment:
`labels` is now a filter type in v6   [reference](https://github.com/openshift/cluster-logging-operator/blob/master/docs/administration/upgrade/v6.0_changes.adoc#filters--pipeline-configuration)
```
spec:
  filters:
  - name: my-labels
    type: openshiftLabels
    openshiftLabels:
      foo: bar
  pipelines:
    filterRefs:
    - my-labels
```

So this should be added as a `filter.type.openshiftLabels` and reference in `pipelines.filterRefs`